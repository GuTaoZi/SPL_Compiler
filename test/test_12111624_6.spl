int main(){
    int a;
    b;
    int c;
    d;
    int e;
    f;
    g;
    
}