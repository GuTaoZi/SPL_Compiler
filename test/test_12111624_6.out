Error type B at Line 5: Missing specifier
Error type B at Line 3: Missing specifier
